languages/zh-TW.js

```diff
@@ -0,0 +1,188 @@
+			INHIBITOR_REQUIRED_SETTINGS: (settings) => `此伺服器缺漏 **${settings.join(', ')}** ${settings.length !== 1 ? '等' : ''}的伺服器設定，因此無法執行該指令。`,
+			INHIBITOR_RUNIN: (types) => `此指令只能在 ${types} 的頻道中使用。`,
+			INHIBITOR_RUNIN_NONE: (name) => `指令 ${name} 並沒有被設定為在任何頻道中執行。`,
+			COMMAND_BLACKLIST_DESCRIPTION: '讓機器人對使用者或伺服器封鎖或取消封鎖。',
+			COMMAND_BLACKLIST_SUCCESS: (usersAdded, usersRemoved, guildsAdded, guildsRemoved) => [
+				usersAdded.length ? `**名單已追加使用者**\n${util.codeBlock('', usersAdded.join(', '))}` : '',
+				usersRemoved.length ? `**名單已移除只用者**\n${util.codeBlock('', usersRemoved.join(', '))}` : '',
+				guildsAdded.length ? `**名單已加入伺服器**\n${util.codeBlock('', guildsAdded.join(', '))}` : '',
+				guildsRemoved.length ? `**名單已移除伺服器**\n${util.codeBlock('', guildsRemoved.join(', '))}` : ''
+			].filter(val => val !== '').join('\n'),
+			COMMAND_EVAL_DESCRIPTION: '直接運行 Javascript。僅限機器人持有者使用。',
+			COMMAND_EVAL_EXTENDEDHELP: [
+				'eval 指令直接運行程式碼，拋出的任何錯誤都會被處理。',
+				'此外還提供旗標以調整一些細項。輸入 --silent, --depth=number 或 --async 來調整輸出格式。',
+				'輸入 --silent 旗標，可以隱藏輸出結果。',
+				'輸入 --depth 旗標並賦予一個數字，可以調整 util.inspect 的 depth。',
+				'輸入 --async 旗標，可以將程式碼包在一個 async function，然後就能隨心所欲使用 await；若你想要有回傳值，就要有關鍵詞 return 。',
+				'輸入 --showHidden 旗標，可以啟用 util.inspect 的 showHidden 項目。',
+				'若輸出結果太長，將會包成一個檔案；機器人沒有 ATTACH_FILES 權限的話，則會直接列在 console 中。'
+			].join('\n'),
+			COMMAND_EVAL_ERROR: (time, output, type) => `**錯誤**:${output}\n**型別**:${type}\n${time}`,
+			COMMAND_EVAL_OUTPUT: (time, output, type) => `**輸出**:${output}\n**型別**:${type}\n${time}`,
+			COMMAND_EVAL_SENDFILE: (time, type) => `輸出結果太長了... 已將之包在一個檔案內了。\n**型別**:${type}\n${time}`,
+			COMMAND_EVAL_SENDCONSOLE: (time, type) => `輸出結果太長了... 已經之直接列在 console 中。\n**型別**:${type}\n${time}`,
+			COMMAND_UNLOAD: (type, name) => `✅ 已卸載 ${type}: ${name}`,
+			COMMAND_UNLOAD_DESCRIPTION: '卸載 klasa piece。',
+			COMMAND_UNLOAD_WARN: '你不會想要卸載那個的，因為屆時你將無法再運行任何指令啟用回去。',
+			COMMAND_TRANSFER_ERROR: '❌ 該檔案已傳輸完畢，或是根本從不存在。',
+			COMMAND_TRANSFER_SUCCESS: (type, name) => `✅ 已成功傳輸 ${type}: ${name}.`,
+			COMMAND_TRANSFER_FAILED: (type, name) => `${type}: ${name} 沒有成功傳輸至客戶端。請查看你的 Console。`,
+			COMMAND_TRANSFER_DESCRIPTION: '傳輸一個 Core piece 至相對應的資料夾。',
+			COMMAND_RELOAD: (type, name, time) => `✅ 已經重新載入 ${type}: ${name}。（歷時：${time}）`,
+			COMMAND_RELOAD_FAILED: (type, name) => `❌ 重新載入失敗 ${type}: ${name}。請查看你的 Console。`,
+			COMMAND_RELOAD_ALL: (type, time) => `✅ 已經重新載入所有的 ${type}。（歷時：${time}）`,
+			COMMAND_RELOAD_EVERYTHING: (time) => `✅ 已經重新載入所有元件。（歷時：${time}）`,
+			COMMAND_RELOAD_DESCRIPTION: '重新載入一個 Klasa piece，或 Klasa 保有的所有 piece。',
+			COMMAND_REBOOT: '重啟中...',
+			COMMAND_REBOOT_DESCRIPTION: '重啟機器人。',
+			COMMAND_LOAD: (time, type, name) => `✅ 已成功載入 ${type}: ${name}。（歷時：${time}）`,
+			COMMAND_LOAD_FAIL: '檔案不存在，或是在讀取你檔案時發生了一個錯誤。請查看你的 Console。',
+			COMMAND_LOAD_ERROR: (type, name, error) => `❌ 載入失敗 ${type}: ${name}。 原因：${util.codeBlock('js', error)}`,
+			COMMAND_LOAD_DESCRIPTION: '藉由機器人直接載入一個 Piece。',
```

Comment:
Is the word `Piece` not translatable? Same for all other occurrences.

Reply:
It can be translated as "碎片",but that way may confuse a user, as Klasa documentation doesn't have the Traditional Chinese version yet. If they search for the term "klasa 碎片", they will get nothing helpful.